src/random.h

```diff
@@ -67,9 +67,21 @@
  * Thread-safe.
  */
 void GetRandBytes(unsigned char* buf, int num) noexcept;
+/** Generate a uniform random integer in the range [0..range). Precondition: range > 0 */
 uint64_t GetRand(uint64_t nMax) noexcept;
```

Comment:
```suggestion
/** Generate a uniform random integer in the range [0..nMax). Precondition: nMax > 0 */
uint64_t GetRand(uint64_t nMax) noexcept;
```

Reply:
Thanks. To not invalidate all the ACKs, I am going to leave this for now.